src/openrct2-ui/windows/Park.cpp

```diff
@@ -1431,7 +1431,7 @@ static void window_park_stats_paint(rct_window* w, rct_drawpixelinfo* dpi)
     y += LIST_ROW_HEIGHT;
 
     // Draw number of rides / attractions
-    if (w->list_information_type != 0xFFFF)
+    if (w->list_information_type != static_cast<uint16_t>(-1))
```

Comment:
It used to be like that, but @Gymnasiast suggested the `0xFFFF`, do we want to go back?